pkg/controller/utils/common/common.go

```diff
@@ -45,3 +46,37 @@ func LookupHost(host string, logger logr.Logger) (string, error) {
 	logger.Info("host resolved", "host", host, "addresses", addresses)
 	return host, nil
 }
+
+const (
+	// OperandImageOpenJDK envvar name for OpenJDK operand image
+	OperandImageOpenJDK = "RELATED_IMAGE_OPENJDK"
```

Comment:
What is motivation for implementing a new env variable?
I'm prefer to move from env variables to the one configuration file in the ConfigMap as describer in #491

Reply:
this because some build process needs those variables (RELATED_IMAGE_*) in the CSV by convention. i.e. they're used to enable installation in disconnected environments

Reply:
Oh, thanks, this is more incompatible with DEFAULT_IMAGE, maybe we need to brake DEFAULT_IMAGE using in the master?